modules/virt-creating-configmap.adoc

```diff
@@ -1,36 +1,111 @@
+. In a web browser, identify the RHV VM operating system by navigating to `\http://<RHV_Manager_FQDN>/ovirt-engine/api/vms/<VM_ID>`. The operating system appears in the `<os>` section of the XML output, as shown in the following example:
 +
 [source,xml]
 ----
 <os>
 ...
-<type>windows_2016x64</type>
+<type>rhel_8x64</type>
 </os>
 ----
 
-. Obtain a list of the common {VirtProductName} operating system templates by running the following command on your {product-title} cluster:
+. To import a Linux VM:
+
+.. Obtain a list of the {product-title} templates:
 +
 ----
 $ oc get templates -n openshift --show-labels | tr ',' '\n' | grep os.template.kubevirt.io | sed -r 's#os.template.kubevirt.io/(.*)=.*#\1#g' | sort -u
 ----
 +
-If the RHV VM operating system does not have a corresponding {VirtProductName} template, you must create a custom ConfigMap.
+The output is similar to the following example:
++
+----
+fedora31
+fedora32
+...
+rhel8.1
+rhel8.2
+...
+----
++
+If the RHV VM operating system appears in this template list, you do not need to create a ConfigMap.
+
+.. If the RHV VM operating system does not appear in the template list, create a ConfigMap for a common template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default <1>
+data:
+  guestos2common: |
+    "Red Hat Enterprise Linux Server": "rhel" <2>
+    "CentOS Linux": "centos"
+    "Fedora": "fedora"
+    "Ubuntu": "ubuntu"
+    "openSUSE": "opensuse"
+  osinfo2common: |
+    "<operating-system>": "<common-template>" <3>
+EOF
+----
+<1> Optional: You can change the value of the `namespace` parameter.
+<2> Specify the Linux operating system.
+<3> Specify the Linux operating system version and the {product-title} common template that it maps to:
+
+* `"rhel_6_9_plus_ppc64": "rhel6.9"`
+* `"rhel_6_ppc64": "rhel6.9"`
+* `"rhel_6": "rhel6.9"`
+* `"rhel_6x64": "rhel6.9"`
+* `"rhel_7_ppc64": "rhel7.7"`
+* `"rhel_7_s390x": "rhel7.7"`
+* `"rhel_7x64": "rhel7.7"`
+* `"rhel_8x64": "rhel8.1"`
+* `"sles_11_ppc64": "opensuse15.0"`
+* `"sles_11": "opensuse15.0"`
+* `"sles_12_s390x": "opensuse15.0"`
+* `"ubuntu_12_04": "ubuntu18.04"`
+* `"ubuntu_12_10": "ubuntu18.04"`
+* `"ubuntu_13_04": "ubuntu18.04"`
+* `"ubuntu_13_10": "ubuntu18.04"`
+* `"ubuntu_14_04_ppc64": "ubuntu18.04"`
+* `"ubuntu_14_04": "ubuntu18.04"`
+* `"ubuntu_16_04_s390x": "ubuntu18.04"`
+
+.. If your operating system does not appear in the common template list, create a ConfigMap to map `other_linux` to the `fedora31` common template:
++
+[source,yaml]
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: os-configmap
+  namespace: default
+data:
+  osinfo2common: |
+    "other_linux": "fedora31"
+EOF
+----
+
+. To import a Windows VM:
 
-. If the RHV VM uses a Windows operating system, create a Windows {VirtProductName} template by running the following command:
+.. Create a Windows template:
```

Comment:
Windows custom template

Reply:
Why is 'custom' necessary? If the user is creating a Windows template, then it is a custom template. Not necessary to say "Windows custom template".

Reply:
yes, you are right.